packages/apps/src/app.plugins.ts

```diff
@@ -29,17 +30,26 @@ export function plugin(this: App, plugin: IPlugin) {
 /**
  * get a plugin
  */
-export function getPlugin(this: App, name: PluginName): IPlugin | undefined {
+export function getPlugin<TPlugin extends IPlugin>(
```

Comment:
a default for the generic of `IPlugin` would help for those that doesn't have a specific type in mind.

Reply:
Check out https://github.com/microsoft/teams.ts/pull/205#discussion_r2080939096